azure-iot-device/azure/iot/device/provisioning/registration_client.py

```diff
@@ -0,0 +1,40 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module provides an abstract interface representing clients which can communicate with the
+Device Provisioning Service.
+"""
+
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class RegistrationClient(object):
```

Comment:
>RegistrationClient [](start = 6, length = 18)

I find this name weird still. Like, it's the Provisioning Library, but it exposes a RegistrationClient?

Reply:
I think the "vetted" name for other SDKs is ProvisioningDeviceClient.

---
In reply to: [277768519](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r277768519) [](ancestors = 277768519)